apps/web/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nikolaik/python-nodejs:python3.10-nodejs16-slim
+FROM nikolaik/python-nodejs:python3.10-nodejs16-alpine
 
 WORKDIR /usr/src/app
```

Comment:
the deps that are used by the web app are: `notification-center, shared`
the dev deps: `dal, testing`
no need to copy all others